WordPress/wordcamp.org
# Prevent users from creating new WordCamp and Meetup posts on central dashboard

Prevent users from creating new WordCamp and Meetup posts on dashboard. In most of the cases, all posts should be created thru the public application forms in order to get all needed information and to initiate the vetting process correctly.

Expectation is made for users with `wordcamp_wrangle_wordcamps` and  `wordcamp_wrangle_meetups` capabilities, as deputies need to create events manually from time to time.

Fixes #728, See #734 and #1041  and #950 (this is retry of it) and #1031 (which reverted 950)

Props @NewYorkerLaura @iandunn 

### How to test the changes in this Pull Request:

1. Log in as user with contributor role
2. Send a WordCamp application via public form https://central.wordcamp.test/wordcamp-organizer-application/
3. Log out and log in as user with admin or deputy role
4. Navigate to https://central.wordcamp.test/wp-admin/edit.php?post_type=wordcamp and you should see the application just sent
5. Navigate to https://central.wordcamp.test/wp-admin/post-new.php?post_type=wordcamp and you should be able to create a new WordCamp
6. Add the user with contributor role to the site
7. On wp-config.php `$wcorg_subroles` array remove `wordcamp_wrangler` and `meetup_wrangler` subroles from contributor role user
8. Log out and log in as user with contributor role
9. Navigate to central dashboard and try to update the WordCamp you send application for
10. Check that you don't see "Add new" button on WordCamp listing
